### PR TITLE
sys/xtimer: add xtimer_set_timeout_flag64()

### PR DESCRIPTION
**Pull request by kaspar030**
This PR adds a 64bit version of xtimer_set_timeout_flag().

### CODE REVIEW

File: sys/include/xtimer.h [hunk 1]
```diff
@@ -448,7 +448,14 @@ int xtimer_mutex_lock_timeout(mutex_t *mutex, uint64_t us);
  * @param[in]   t       timer struct to use
```
Comment (gebart) [inline sys/include/xtimer.h]: you forgot to update the param name in the doxygen comment

Comment (kaspar030) [reply sys/include/xtimer.h]: fixed

File: sys/include/xtimer.h [hunk 2]
```diff
@@ -445,10 +445,17 @@ int xtimer_mutex_lock_timeout(mutex_t *mutex, uint64_t us);
  * This function will set THREAD_FLAG_TIMEOUT on the current thread after @p
  * timeout usec have passed.
  *
- * @param[in]   t       timer struct to use
+ * @param[in]   timer   timer struct to use
```
Comment (smlng) [inline sys/include/xtimer.h]: why renaming this, when its actual implementation still uses `t`?

Comment (miri64) [reply sys/include/xtimer.h]: Ping @kaspar030?

**Comment by smlng** [comment]
ping

**Comment by kaspar030** [comment]
> As an alternative we could close this one, too.

I'm for closing. Feel free to adopt if a need arises.

**Pull request closed without merging**